services/web/server/src/simcore_service_webserver/login/settings.py

```diff
@@ -51,6 +51,11 @@ class LoginSettings(BaseCustomSettings):
         description="If true, it enables two-factor authentication (2FA)",
     )
 
+    LOGIN_PASSWORD_MIN_LENGTH: PositiveInt = Field(
+        default=12,
```

Comment:
May I ask to put the `LOGIN_PASSWORD_MIN_LENGTH` in the .env-devel file default to 3?
Usually people use short passwords when developing.